specification/attributes/column_naming_convention.md

```diff
@@ -0,0 +1,34 @@
+# Column naming convention
+
+Column IDs provided in billing datasets following a consistent naming convention reduces friction for FinOps
+practitioners that consume the datasets for analysis, reporting, and other use cases.
+
+All columns defined in the FOCUS specification MUST follow the naming requirements listed below. Provider generated
```

Comment:
Isn't "FOCUS specification" redundant? We should take an official position, as others have.
```suggestion
All columns defined in FOCUS MUST follow the naming requirements listed below. Provider generated
```

Reply:
I think we mentioned in the docs that both are correct (even though the last S is for specification).. Updated